src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const filteredStylesList = sourceString
+    .split(';')
+    .filter((style) => /\S/.test(style));
+  const resultObject = {};
+
+  for (const styleItem of filteredStylesList) {
+    const keyValue = styleItem.split(':').map((item) => item.trim());
+    const [property, value] = [...keyValue];
+
+    resultObject[property] = value;
+  }
+
+  return resultObject;
```

Comment:
The function `convertToObject` does not follow the checklist item 5, which advises against using loops. Instead of the `for...of` loop, consider using array iteration methods like `reduce` to transform the array of style strings into an object.